02-analytics/technical.md

```diff
@@ -0,0 +1,377 @@
+# Technical details for the tracking plan
+
+## Implementation notes
+
+The analytics should be collected (unless otherwise noted), even when the client
+is offline. The data should be cached as far as possible and sent to the server
+when the device is back online. If caching more data becomes impossible, the oldest
+cached data should be deleted. The deletion is discussed further later in the doc.
+
+## Common data
+
+The data specified below should be sent with every event, unless otherwise noted. 
+If it is not possible for all events then that should be noted in this document.
+
+In general it is better to omit data (if for example it is not accessible on some
+os version), and send what you can.
+
+Note that this is for both APP and Web so some things may be possible on one 
+platform and not on the other.
+
+### Common data table
+
+| Data                | Name           | Comments                                      |
+|---------------------|----------------|-----------------------------------------------|
+| OS                  | os             | Automatically managed by SDKs                 |
+| OS Language         | locale         | Automatically managed by SDKs                 |
+| Network Info        | networkInfo    | Automatically managed by SDKs                 |
+| Device Info         | deviceInfo     | Automatically managed by SDKs                 |
+| Anonymous ID        | anonymousId    | Automatically managed by SDKs                 |
+| Channel             | channel        | Automatically managed by SDKs                 |
+| Timezone            | timezone       | Automatically managed by SDKs                 |
+| Screen data         | screen         | Automatically managed by SDKs                 |
+| App Language        | appLanguage    |                                               |
+| Release Version     | releaseVersion | App version, or build number/git hash for web |
+| Person analytics ID | presonId       | For logged in users, see below                |
+
+### Person analytics ID
+
+You can get this ID by sending a token to TODO. The returned value is an hash
+of the userID + a secret. This ensures that we are able to link all data to one
+user from different systems, while preventing anyone from being able to reverse
+the process or generate analytics IDs if they do not posses the secret.
+
+## Identify
+
+*When*: On login + When you request a new refresh token from Auth0
+*Reason*: Being able to connect anonymous events to a user
```

Comment:
If you can connect events to a user, are events anonymous?

Reply:
1. When you start and are not logged in you are anonymous
2. You log in and we assign you a ID that can not (without admin access) be linked to a PERSON, but you are considered and "identified user"
3. This way we can link events for a user (not person) across devices

I will reword this statement, but I hope the explanation makes it more clear.

Reply:
OK, thank you for clarifying anonymous thing here.